packages/js-sdk/src/sandbox/sandboxApi.ts

```diff
@@ -66,6 +66,59 @@ export interface SandboxInfo {
   endAt: Date
 }
 
+export interface ListedSandbox {
+  /**
+   * Sandbox ID.
+   */
+  sandboxId: string
+
+  /**
+   * Template ID alias.
+   */
+  alias?: string;
+
+  /**
+   * Template ID.
+   */
+  templateId: string;
+
+  /**
+   * Client ID.
+   */
+  clientId: string;
```

Comment:
not sure if we need to expose this, can we at least mark it as deprecated for now?